test/unit/cfg_errors_test.lua

```diff
@@ -320,12 +320,14 @@ end
 
 g.test_positive = function()
     g.mock_membership()
+    membership.broadcast = function() error('Forbidden', 0) end
```

Comment:
I think error message should contain "Membership broadcast is forbidden" but not just "Forbidden"